CredentialProvider.Microsoft/CredentialProviders/Vsts/AdalTokenProvider.cs

```diff
@@ -74,11 +74,11 @@ public async Task<IAdalToken> AcquireTokenSilentlyAsync(CancellationToken cancel
             {
                 return null;
             }
-        }
-
-#pragma warning disable CS1998 // Async method lacks 'await' operators and will run synchronously
-        public async Task<IAdalToken> AcquireTokenWithUI(CancellationToken cancellationToken)
-#pragma warning restore CS1998 // Async method lacks 'await' operators and will run synchronously
+        }
+
+#pragma warning disable CS1998 // Async method lacks 'await' operators and will run synchronously
```

Comment:
nit: there is no need to disable this warning. Just remove the `async` keyword. It won't affect inheritance.

Reply:
This is because the method has pragmas between .NET Framework and .NET Core, one of which has await and the other doesn't. Alternatively the entire method could be pragma'd instead of just the impl. Hoping to remove all this once MSAL is rolled out and made the default.